sw/rust/ibex-demo-system-hal/src/spi.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use embedded_hal::blocking;
+use embedded_hal::digital::v2::OutputPin;
+use embedded_hal::spi;
+
+use ibex_demo_system_pac::SPI0;
+
+pub struct Spi<T: OutputPin> {
```

Comment:
You might want to name the generic type as `CS: OutputPin`

Reply:
Done